gotocompany/optimus/core/v1beta1/job_run.proto

```diff
@@ -40,6 +40,12 @@ service JobRunService {
       get: "/v1beta1/project/{project_name}/job/{job_name}/run"
     };
   }
+  // JobRunList returns the current and past run status of jobs on a given range
+  rpc GetLatestJobRuns(GetLatestJobRunsRequest) returns (GetLatestJobRunsResponse) {
```

Comment:
I'm thinking of keeping the number of rpc methods to less by merging the scope of this rpc method with the existing rpc method to get jobruns, should we look into where all we are using the existing method & try to unify them instead.